agent/grpc-external/services/dataplane/get_envoy_bootstrap_params.go

```diff
@@ -59,7 +59,6 @@ func (s *Server) GetEnvoyBootstrapParams(ctx context.Context, req *pbdataplane.G
 			Tenancy: &pbresource.Tenancy{
 				Namespace: req.Namespace,
 				Partition: req.Partition,
-				PeerName:  "local",
```

Comment:
Should we keep it given that k8s sets it?

Reply:
It does pass in the k8s even with this removed and I'm assuming that's because the resource api is defaulting the peer name for reads also?

Reply:
do the explicit examples work?  this was required when RB and I paired on getting the v2 examples working with this computed resources work.

Reply:
I just confirmed they still work.  disregard.